src/main/resources/hudson/plugins/jobConfigHistory/JobConfigHistoryRootAction/resource.js

```diff
@@ -0,0 +1,26 @@
+window.addEventListener("DOMContentLoaded", () => {
+    hljs.initHighlightingOnLoad();
+
+    var showVersionDiffsJs = true;
+
+    function toggleShowHideVersionDiffsJs(button, hideText, showText) {
+        showVersionDiffsJs = !showVersionDiffsJs;
+        if (showVersionDiffsJs === true) {
+            document.getElementById("tbody_versionDiffsShown").style.display = '';
+            document.getElementById("tbody_versionDiffsHidden").style.display = 'none';
+
+            button.value = hideText;
+        } else {
+            document.getElementById("tbody_versionDiffsShown").style.display = 'none';
+            document.getElementById("tbody_versionDiffsHidden").style.display = '';
+
+            button.value = showText;
+        }
+    }
```

Comment:
```suggestion
    var showVersionDiffsJs = true;

    function toggleShowHideVersionDiffsJs(hideText, showText) {
        showVersionDiffsJs = !showVersionDiffsJs;
        if (showVersionDiffsJs === true) {
            document.getElementById("tbody_versionDiffsShown").style.display = '';
            document.getElementById("tbody_versionDiffsHidden").style.display = 'none';

            document.getElementById("showHideVersionDiffsJsButton").value = hideText;
        } else {
            document.getElementById("tbody_versionDiffsShown").style.display = 'none';
            document.getElementById("tbody_versionDiffsHidden").style.display = '';

            document.getElementById("showHideVersionDiffsJsButton").value = showText;
        }
    }
```

Diff with the old script.